bot/start.ts

```diff
@@ -565,11 +570,11 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
           const community = await Community.findById(
             ctx.admin.default_community_id
           );
-          community.banned_users.push({
+          community?.banned_users.push({
             id: user._id,
             username: user.username,
           });
-          await community.save();
+          await community?.save();
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program